src/main/java/com/powsybl/openloadflow/ac/equations/AbstractHvdcAcEmulationFlowEquationTerm.java

```diff
@@ -55,6 +58,20 @@ protected double rawP(double p0, double k, double ph1, double ph2) {
         return p0 + k * (ph1 - ph2);
     }
 
+    protected double rawPWithLoss(double rawP, double loss1, double loss2, double v, double r) {
```

Comment:
We already have in `powsybl-core` the method in `HvdcUtils.getHvdcLineLosses(...)`. Can we imagine reusing it here because it is what we use when the hvdc is operated in active power set point mode.